sdk/core/src/services/BloomVaultStorageService.ts

```diff
@@ -233,15 +242,32 @@ export default class BloomVaultStorageService {
   }
 
   public async searchCredentials(storageRegion: string, types?: string[][]): Promise<any[]> {
-    const accessToken = await this._authorizeVcVault(storageRegion)
+    let migration: any = { status: undefined }
+    try {
+      migration = await this._migrationHelper.getMigrationStatus()
+    } catch (err) {
+      console.log('@@@Vault-migration-service migration status call ends with error: ', err)
+    }
 
-    const credentials = await this._fetchAllDecryptedCredentials(accessToken, storageRegion)
+    if (migration.status === 'done') {
+      return []
+    } else {
```

Comment:
minor: no need for an else statement here it makes the code harder to read. See https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 for the reasoning.

Reply:
ok, will do. thanks